cumulus_library/cli_parser.py

```diff
@@ -182,6 +182,15 @@ def get_parser() -> argparse.ArgumentParser:
         ),
         dest="stats_build",
     )
+    build.add_argument(
+        "--umls",
+        help="An API Key for the UMLS API",
+    )
+    build.add_argument(
+        "--replace-existing",
```

Comment:
(Though `--force` may get overloaded once we have two kinds of things we want to force through, so it's not necessarily a good idea, just _an_ idea.)

Reply:
this is so weird because it's only affecting a slice of the total state. lemme sleep on it, i'll try to get something a bit more nuanced, but it may just be `--replace-existing-uploads` - :hopefully: this is developer only, though there are edge cases for users

Reply:
`--force-uploads` splits the difference